src/cascadia/TerminalApp/PaletteItemTemplateSelector.h

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
```

Comment:
nit: mind adding an abstract here? Maybe mention the difference between the two templates and when to use one over the other.

Reply:
Added method documentation instead